op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -0,0 +1,104 @@
+package loadtest
+
+import (
+	"math/rand"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/bindings"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/contracts/constants"
+	"github.com/ethereum-optimism/optimism/op-acceptance-tests/tests/interop"
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/txintent"
+	"github.com/ethereum-optimism/optimism/op-service/txplan"
+	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+var SimpleInterop presets.TestSetup[*presets.SimpleInterop]
+
+// TestMain creates the test-setups against the shared backend
+func TestMain(m *testing.M) {
+	presets.DoMain(m, presets.NewSimpleInterop(&SimpleInterop))
+}
+
+// TODO: this helper exists in some form in at least three places:
+// 1. here
+// 2. op-acceptance-tests/tests/interop/interop_test_helpers.go
+// 3. op-e2e/actions/interop/interop_txplan_test.go
+// Should we make a single helper in devnet-sdk/contracts?
+func deployContract(t devtest.T, bin string, opt txplan.Option) common.Address {
+	tx := txplan.NewPlannedTx(txplan.WithData(common.FromHex(bin)), opt)
+	_, err := tx.Success.Eval(t.Ctx())
+	t.Require().NoError(err)
+	deployReceipt, err := tx.Included.Get()
+	t.Require().NoError(err)
+	return deployReceipt.ContractAddress
+}
+
+func TestExecFromSameAddressInALoop(gt *testing.T) {
+	t := devtest.ParallelT(gt)
+	sys := SimpleInterop(t)
+	eoaA := sys.FunderA.NewFundedEOA(eth.Ether(10))
+	eoaB := sys.FunderB.NewFundedEOA(eth.Ether(10))
+
+	eventLoggerAddress := deployContract(t, bindings.EventloggerBin, eoaA.Plan())
+
+	// Create a transaction that emits numMsgs logs in a single transaction using multicall and the event logger.
+	const numMsgs = 275 // About the max number of msgs we can create before hitting tx size limits.
+	initMsgsTx := txintent.NewIntent[*txintent.MultiTrigger, *txintent.InteropOutput](eoaA.Plan())
+	initCalls := make([]txintent.Call, 0, numMsgs)
+	rng := rand.New(rand.NewSource(1234))
+	for range numMsgs {
+		initCalls = append(initCalls, interop.RandomInitTrigger(rng, eventLoggerAddress, rng.Intn(5), rng.Intn(10)))
+	}
+	initMsgsTx.Content.Set(&txintent.MultiTrigger{
+		Emitter: constants.MultiCall3,
+		Calls:   initCalls,
+	})
+
+	_, receipt, err := interop.ValidateEveryMessage(t.Ctx(), eoaB.Plan(), initMsgsTx)
+	t.Require().NoError(err)
+	t.Require().Len(receipt.Logs, numMsgs)
+}
+
+// Spam txs that take near the gas limit (leaving room for system tx), but have invalid exec msg.
+func TestLargeTxWithInvalidExecMsg(gt *testing.T) {
```

Comment:
Spoke in person about this -- let's strike this test for now, in favor of more straightforward spammy tests. In the meantime we should decide what this test is meant for:

- Invalid messages won't make it to the builder, because ingress / Tx pipeline will drop it early
- Just burning gas isn't a good representation of workload, which we may want if this is a load test against the network in total (ie: do a bunch of disk operations to use the gas because just burning it isn't using it)
- What does "Large" mean here anyway? Large number of Executing Messages? Gas?